test/integration/test_kubernetes.py

```diff
@@ -81,45 +75,35 @@ def test_03_k8s_status(self, mocked_sys_exit):
                                 content_type="application/x-amz-json-1.1")
         with captured_output() as (out, err):
             kubernetes.k8s_status(TEST_KUBE_CONTEXT, True, False, False, False, filename=self.__class__.kubernetes_yml)
-            mocked_sys_exit.assert_called_once_with(0)
             self.assertIn('Resources', out.getvalue().strip())
             # self.assertIn('deployment.apps/hello-web', out.getvalue().strip())
             # self.assertIn('service/hello-web', out.getvalue().strip())
```

Comment:
Why is there all this commented out code in our tests?

Reply:
a [5 year old commit](https://github.com/artsy/hokusai/commit/33b1d6f3f4597035e3c6d1d65dd793d5ee086d68). sounds like we wanted these assertions but they caused some stdout/err problems.

Reply:
i am actually also working on creating a comprehensive set of integration tests. so i'll likely run into that problem and i'll address it then.